packages/react-composites/src/composites/ChatComposite/FileDownloadErrorBar.tsx

```diff
@@ -3,35 +3,34 @@
 
 import { IIconProps, MessageBar, MessageBarType, Stack } from '@fluentui/react';
 import React from 'react';
+import { ChatCompositeIcon } from '../common/icons';
 
 /**
  * @private
  */
 export interface FileDownloadErrorBarProps {
   /**callback to dismiss the download error message */
-  onDismissDownloadErrorMessage?: () => void;
+  onDismissDownloadErrorMessage: () => void;
   /** Error message to render */
-  fileDownloadErrorMessage?: string;
+  fileDownloadErrorMessage: string;
 }
 
 /**
  * @private
  */
 export const FileDownloadErrorBar = (props: FileDownloadErrorBarProps): JSX.Element => {
   const { fileDownloadErrorMessage, onDismissDownloadErrorMessage } = props;
-  const messageBarIconProps = (): IIconProps | undefined => {
-    const iconName = { iconName: 'ProtectedDocument' };
-    return iconName;
-  };
-  if (fileDownloadErrorMessage && fileDownloadErrorMessage !== '') {
+  const messageBarIconProps = { iconName: 'ProtectedDocument' };
```

Comment:
This const will still create a new object on each render pass and cause MessageBar to be rerendered each time. I mean to add a file-level constant.